README.md

```diff
@@ -151,6 +146,13 @@ No tests here...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Join Tables]**
+ 
```

Comment:
- [ ] Good job adding the `Future Features` section, but please add a corresponding LINK in the table of contents for the future features section as this is the given template design. This will also help users navigate your readme file easily. 
#### Please note that care should be taken to make the readme file as professional as possible, potential recruters may view your projects and if they see a poorly documented project, they may get a wrong impression about your attention to detail. Thats why we emphasize the readme be professional.